router/smoke/openshift_2/conftest.py

```diff
@@ -29,7 +29,10 @@ def pytest_addoption(parser):
     parser.addoption("--cluster", action="append", required=True,
                      help="Openshift clusters IP where routers is deployed")
 
-    parser.addoption("--msg-length", action="append", required=False, default=['256'],
+    parser.addoption("--token", action="append", required=True,
+                     help="Token to authenticate with OCP same order as --cluster")
```

Comment:
Tokens are needed, because when we execute "oc scale" (for example) we must be sure we are doing it in the correct instance.

Reply:
Okay, I hoped that we can use execution on the "oc cluster node" -> not the local one.

Reply:
I believe we can, if we define the correct ansible_host on the inventory file.
But anyway the distinct tokens remain necessary, unless we are planning to login as system:admin. Maybe we should try it later and see how it goes.